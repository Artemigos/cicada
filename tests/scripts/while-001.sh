echo hi
counter=17
while echo "$counter" | grep -iq "^1.$"
    echo "counter = $counter"
    counter=$(expr $counter + 1)
done
echo bye
