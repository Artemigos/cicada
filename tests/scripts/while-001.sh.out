hi
counter = 17
counter = 18
counter = 19
bye
